### DISCUSSION TITLE
DAI 2024-2025 - Practical work 1 - David Schildböck and Arno Tribolet

### DISCUSSION THREAD

**Discussion by nyxhawk** [heig-vd-dai-course/heig-vd-dai-course#352] [Show and tell]
A CLI tool for cryptographic operations, allowing users to encrypt and decrypt files using multiple encryption algorithms.

**Comment by ludelafo** [comment]
Looks good to us, thank you!

**Comment by arnoheigvd** [comment]
Our project is finished
Repository [link](https://github.com/arnoheigvd/DAI_labo1_Schildbock_Tribolet)  
Commit [d8fb107](https://github.com/arnoheigvd/DAI_labo1_Schildbock_Tribolet/commit/d8fb107fb1a91d1c3d9d37cee71a2480fc3746bf)

**Reply by nyxhawk** [reply]
A [tag](https://github.com/arnoheigvd/DAI_labo1_Schildbock_Tribolet/releases/tag/rendu) has also been created if needed !
